src/validation.cpp

```diff
@@ -588,28 +588,28 @@ static bool AcceptToMemoryPoolWorker(const CChainParams& chainparams, CTxMemPool
 
     // Coinbase is only valid in a block, not as a loose transaction
     if (tx.IsCoinBase())
-        return state.Invalid(TxValidationResult::CONSENSUS, false, REJECT_INVALID, "coinbase");
+        return state.Invalid(TxValidationResult::CONSENSUS, false, "coinbase");
 
     // Rather not work on nonstandard transactions (unless -testnet/-regtest)
-    std::string reason;
+    std::string reason {};
```

Comment:
In commit "[validation] Remove Reject Code from ValidationState" (a3076817588a642b4de107dc96a76ea8939070df)

Unintentional change? Doesn't serve an obvious purpose.

Reply:
No longer part of this PR.